prow/config.yaml

```diff
@@ -12,6 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+tide:
+  merge_method:
+    GoogleCloudPlatform/osconfig: squash
```

Comment:
Please make sure that the GH repo settings are configured to allow squash merges in the repo.